SPECS/git-lfs/git-lfs.spec

```diff
@@ -1,8 +1,8 @@
 %global debug_package %{nil}
 Summary:       Git extension for versioning large files
 Name:          git-lfs
-Version:       3.1.4
-Release:       17%{?dist}
+Version:       3.4.1
```

Comment:
The PR:
- https://github.com/microsoft/azurelinux/pull/8871

supersedes this PR and maybe we might as well close it.

Reply:
Yep, looks like it does, thank you. Will close